### ISSUE TITLE
ComponentListener fails to compile when upgrading exo player

### ISSUE THREAD

**Issue by dpreussler** [googlesamples/androidtv-Leanback#107]
when upgrading to Exoplayer 2.6(.1)
```ComponentListener```
fails to compile as ```onPositionDiscontinuity``` has changed signature and new additional callbacks are introduced

**Comment by benbaxter** [comment]
I just upgraded the sample and will push the update shortly. When I updated to 2.6.1, I did get some errors in Android Studio when syncing gradle and building. I did Build -> Clean Project it worked just fine afterwards.

I see that ExoPlayer had a [commit][1] that added the discontinuity reason to the method. I wonder if that is causing issues for you.

[1]
 (https://github.com/google/ExoPlayer/commit/7d59383cc442d8e443e08732c985b886f8d033a0#diff-33191a23fa8d8d2f7a14c9e6ea7324b0)
